test/application_system_test_case.rb

```diff
@@ -59,11 +59,11 @@ def teardown
     Capybara.use_default_driver
   end
 
-  driven_by :selenium, using: :chrome, screen_size: [1400, 1400]
-  # driven_by :selenium, using: :headless_chrome do |driver_option|
-  #   # Without this argument, Chrome cannot be started in Docker
-  #   driver_option.add_argument('no-sandbox')
-  # end
+  # driven_by :selenium, using: :chrome, screen_size: [1400, 1400]
+  driven_by :selenium, using: :headless_chrome do |driver_option|
+    # Without this argument, Chrome cannot be started in Docker
+    driver_option.add_argument('no-sandbox')
```

Comment:
Yep. We often selenum with firefox btw. Might be worthwhile too, should you run into more chrome-ish issues.